main.js

```diff
@@ -0,0 +1,28 @@
+const header = document.querySelector("li");
+const readTime = document.querySelectorAll("span");
+const articles = document.querySelectorAll("article");
+
+// Removes only the first li, admittedly.
+if (header.textContent.includes("All Posts")) header.remove();
```

Comment:
Could probably just do `header.remove()` if you know the first element is going to be the `All Posts` link.